.github/workflows/tests.jsonnet

```diff
@@ -220,6 +221,40 @@ local install_x86_artifacts = {
   |||,
 };
 
+local download_x86_pro_artifacts = {
+  uses: 'actions/download-artifact@v3',
+  with: {
+    name: core_pro_x86.export.artifact_name,
+  },
+};
+local install_x86_pro_artifacts = {
+  name: 'Install pro artifacts',
+  run: |||
+    tar xf ocaml-build-artifacts.tgz
+    sudo cp ocaml-build-artifacts/bin/* /usr/bin
+
+    # The version is stored in cli/src/semgrep/__init__.py
+    # and the file content looks like
+    #   __VERSION__ = "x.xx.x"
+    # so we try to get the x.xx.x part from that file below.
+
+    # First, read the file content
+    version=$(<cli/src/semgrep/__init__.py)
+
+    # Next, we remove the prefix that leads up to the number.
+    version="${version##__VERSION__ = \"}"
+
+    # Finally, we remove the final quote suffix.
+    version="${version%*\"}"
+
+    # By writing the version number to this file and putting it
+    # in the same directory as semgrep-core-proprietary,
+    # the binary will pass the cli's checks and can be run.
+    echo "$version" > pro-installed-by.txt
+    sudo cp pro-installed-by.txt /usr/bin
+  |||,
+};
```

Comment:
Intuitively it seems like it would be easier to use `--custom-binary` like we do in the `semgrep-proprietary` tests? If this is actually hard for some reason I think a comment would be nice to share the knowledge

Reply:
Oooh, I did not know about `--custom-binary`. I'll try and report back!